Doc/library/sqlite3.rst

```diff
@@ -748,6 +748,41 @@ Connection Objects
       .. versionadded:: 3.11
 
 
+   .. method:: serialize(*, name="main")
+
+      This method serializes a database into a :class:`bytes` sequence.  For an
```

Comment:
```suggestion
      This method serializes a database into a :class:`bytes` object.  For an
```
Your wording sounds like it returns a list of bytes objects.

Reply:
Thanks!